testsuite/MDAnalysisTests/data/atomtypes.itp

```diff
@@ -0,0 +1,28 @@
+[ defaults ]
+; nbfunc        comb-rule       gen-pairs       fudgeLJ fudgeQQ
+  1             2               no              1.0     1.0
+
+
+[ atomtypes ]
+;name  at.num  mass        charge          ptype       sigma           epsilon
+A                    10.086     2.000        A       0.356359487256  0.209200
+B    BB   13         20.989    -3.000        A       0.374177461619  0.209200
```

Comment:
What does the BB mean here?

Ah, I see it's a bonded type -- could you update the heading accordingly?

Reply:
Sorry, forgot that while changing the test.